packages/wallet-connectors/src/WalletConnection.ts

```diff
@@ -69,11 +69,17 @@ export function typeSchema(schema: Buffer): TypeSchema {
     };
 }
 
-function schemaAsBuffer(schemaBase64: string) {
+export function schemaAsBuffer(schemaBase64: string) {
+    let unpaddedLen = schemaBase64.length;
+    if (schemaBase64.charAt(unpaddedLen - 1) === '=') {
+        unpaddedLen--;
+    }
+    if (schemaBase64.charAt(unpaddedLen - 1) === '=') {
+        unpaddedLen--;
+    }
```

Comment:
Nit: Might as well move the second `if` inside the former to avoid duplicate check when there's no padding. It's a micro optimization and the runtime probably even does it automatically so it's up to you...